de.dlr.sc.virsat.fdir.core/src/de/dlr/sc/virsat/fdir/core/matrix/Matrix.java

```diff
@@ -161,4 +161,15 @@ public void setStatePredRates(double[][] statePredRates) {
 		this.statePredRates = statePredRates;
 	}
 
-}
\ No newline at end of file
+	@Override

```

Comment:
Is the Matrix class still being used? Looks like we have duplicate code with the matrix factory code.

Reply:
Yes, its duplicate. But some visit methods still use this code. This will be phased out eventually. But for now its still needed.